### PR TITLE
📖 Introduce trigger phrases for e2e test in README

### PR DESCRIPTION
**Pull request by namnx228**
This PR documents e2e trigger phases to the README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,3 +100,11 @@ deployment process.
 ## Architecture
 
 The architecture with the components involved is documented [here](docs/architecture.md)
+
+## E2E test
+To trigger e2e test a PR, use these phases:
+- v1b1 e2e test on Ubuntu: /test-v1b1-e2e
+- v1b1 e2e test on Centos: /test-v1b1-centos-e2e
+- v1a5 e2e test on Ubuntu: /test-v1a1-e2e
```
Comment (kashifest) [inline README.md:108]: a1?

Comment (namnx228) [reply README.md]: Ah, good catch!!! Fixed.

**Review by fmuyassarov — COMMENTED** [review]
/lgtm

**Review by kashifest — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -100,3 +100,15 @@ deployment process.
...
+
+To trigger e2e test on a PR, use these phrases:
+
+- v1b1 e2e test on Ubuntu: /test-v1b1-e2e
+- v1b1 e2e test on Centos: /test-v1b1-centos-e2e
+- v1a5 e2e test on Ubuntu: /test-v1a5-e2e
+- v1a5 e2e test on CentOS: /test-v1a5-centos-e2e
+
+More info about e2e test can be found [here](docs/e2e-test.md)
+
```
Comment (furkatgofurov7) [inline README.md:114]: Maybe it is a bit personal opinion, you might consider it or not (markdown should be happy if you use it, checked locally):
```suggestion

## E2E test

To trigger e2e tests on a PR, please use following phrases:

On main branch:

- **/test-v1b1-e2e** run v1b1 e2e tests on Ubuntu
- **/test-v1b1-centos-e2e** run v1b1 e2e tests on CentOS

release-0.5 branch:

- **/test-v1a5-e2e** run v1a5 e2e tests on Ubuntu
- **/test-v1a5-centos-e2e** run v1a5 e2e tests on CentOS

 More info about e2e test can be found [here](docs/e2e-test.md)

```

**Review by furkatgofurov7 — COMMENTED** [review]
Thanks, looks fine, suggestion inline:

**Review by furkatgofurov7 — COMMENTED** [review]
/lgtm

**Pull request merged**